clicache/src/native_shared_ptr.hpp

```diff
@@ -35,7 +35,8 @@ namespace Apache
         std::shared_ptr<_T>* ptr;
 
       public:
-        native_shared_ptr(const std::shared_ptr<_T>& ptr) : ptr(new std::shared_ptr<_T>(ptr)) {}
+        native_shared_ptr(const std::shared_ptr<_T>& ptr) : ptr(new std::shared_ptr<_T>(ptr)) {
```

Comment:
Intentional formatting change? Almost all our empty methods are `{}` on the same line.

Reply:
Unintentional.